apps/builder/app/shared/instance-utils.test.ts

```diff
@@ -0,0 +1,242 @@
+import { describe, it, expect } from "@jest/globals";
+import { defaultMetas, type WsComponentMeta } from "@webstudio-is/react-sdk";
+import type { Instance } from "@webstudio-is/project-build";
+import {
+  findClosestDroppableComponentIndex,
+  findClosestDroppableTarget,
+} from "./instance-utils";
+
+const defaultMetasMap = new Map(Object.entries(defaultMetas));
+
+describe("find closest droppable component index", () => {
+  const createFakeComponentMetas = (
+    itemMeta: Partial<WsComponentMeta>,
+    anotherItemMeta?: Partial<WsComponentMeta>
+  ): Map<string, WsComponentMeta> => {
+    const base = {
+      label: "",
+      Icon: () => null,
+    };
+    const configs = {
+      Item: { ...base, type: "container", ...itemMeta },
+      AnotherItem: { ...base, type: "container", ...anotherItemMeta },
+      Bold: { ...base, type: "rich-text-child" },
+      TextBlock: { ...base, type: "rich-text" },
+      Form: { ...base, type: "container" },
+      Box: { ...base, type: "container" },
+      Div: { ...base, type: "container" },
+      Body: { ...base, type: "container" },
+    } as const;
+    return new Map(Object.entries(configs)) as Map<string, WsComponentMeta>;
+  };
+
+  it("finds container", () => {
```

Comment:
we use "test()" everywhere

Reply:
describe/it usually goes in pair. Here have 2 describe.

Reply:
that's fine, test() is the same as it(), having both is problematic, brings inconsistencies, you will end up having both all over the place. Ideally we need to even disable it() somehow

Reply:
there are some good answers, specifically this one: https://stackoverflow.com/questions/45778192/what-is-the-difference-between-it-and-test-in-jest#answer-56072272

tldr: it's just about semantics, test() is shorter ... I found in the past `it()` made certain things awkward to express in english and people don't use it properly over time, where test() doesn't enforce semantics

Reply:
Sure here used describe to group tests per utilities.